android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeInstance.kt

```diff
@@ -94,6 +96,16 @@ object CustomerIOReactNativeInstance {
                 config?.getProperty<Boolean>(Keys.Config.AUTO_TRACK_PUSH_EVENTS)?.let { value ->
                     setAutoTrackPushEvents(autoTrackPushEvents = value)
                 }
+                config?.getProperty<String>(Keys.Config.PUSH_CLICK_BEHAVIOR)
+                    ?.takeIfNotBlank()
+                    ?.let { value ->
+                        val behavior = kotlin.runCatching {
+                            enumValueOf<PushClickBehavior>(value)
+                        }.getOrNull()
+                        if (behavior != null) {
+                            setPushClickBehavior(pushClickBehavior = behavior)
+                        }
+                    }
```

Comment:
```suggestion
config?.getProperty<String>(Keys.Config.PUSH_CLICK_BEHAVIOR)?.let { value ->
  val behavior = enumValueOf<PushClickBehavior>(value) ?: return
  setPushClickBehavior(pushClickBehavior = behavior)
}
```

Reply:
better readibility.

Reply:
`enumValueOf` will actually throw exception if the value isn't present in Enum. Wrapping it inside `runCatching` and `getOrNull` guarantees it won't break with unexpected values.